### PR TITLE
Logout functionality

### LINKED ISSUE #237: Log out is not working
**Describe the bug**
I login as stakeholder and found that logout is not working

**Expected behavior**
After clicking logout it should be redirected to login page or home page

**Screenshots**

https://github.com/user-attachments/assets/6520497b-b360-41e5-a62f-11b10ab13a92

**Additional context**

if this issue is previously not addressed i want to work on this

### PR DESCRIPTION
**Pull request by MrunalKashid02**
## 🔨 Changes Made

- Added Logout functionality for stakeholder and student
- 
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [ ] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [x] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

- Fixes #237 

## 📸 Screenshots (if applicable)

https://github.com/user-attachments/assets/c5d3643f-5313-400e-b7d4-e16bed221246

https://github.com/user-attachments/assets/6f32fb91-73c7-48a9-a23e-31cb64b9852a

---

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

File: public/dashboard.html [hunk 1]
```diff
@@ -235,27 +238,16 @@ <h3 style="margin-left:3vw;font-size: 22px;">Edit Profile</h3>
...
-                } else {
-                    console.error("Logout failed:", response.statusText);
-                }
-            } catch (error) {
-                console.error("Logout error:", error);
-            }
+        const handleLogout =  () => {
+            const result1 = document.getElementById('box1');
+            result1.style.display = 'block';
+            localStorage.clear();
```
Comment (Harshdev098) [inline public/dashboard.html:244]: Instead of using clear function use removeItem for removing the token

File: public/dashboard.html [hunk 2]
```diff
@@ -137,6 +137,9 @@ <h3 style="margin-left:3vw;font-size: 22px;">Edit Profile</h3>
                 </div>
             </div>
         </div>
+        <div id="box1">
+            <p>Logout Successfully...</p>
```
Comment (Harshdev098) [inline public/dashboard.html:141]: Also increase the font-size of the message

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by Harshdev098** [comment]
I have commited the changes, resolve branch conflicts!

File: public/dashboard.html (hunk 1 above)
Comment (MrunalKashid02) [reply public/dashboard.html]: I have check by updating branch someone already done this change

File: public/dashboard.html (hunk 2 above)
Comment (MrunalKashid02) [reply public/dashboard.html:141]: i have increase a font size and push the changes.

**Comment by Harshdev098** [comment]
The branch have conflicts resolve them please!

**Comment by MrunalKashid02** [comment]
I have resolve merge conflicts

**Pull request merged**